.github/workflows/test.yml

```diff
@@ -67,3 +67,42 @@ jobs:
           FOLDER: D3DB
           SKIP_EMPTY_COMMITS: true
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  build-website:
+    name: Run D3 Build of website for github pages
+    runs-on: ubuntu-latest
+    # d3build is slow, so only run if lint passes
+    needs: [pre-commit, d3lint]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install graphviz
+        run:  sudo apt-get update && sudo apt install graphviz
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+          cache: 'pip' # caching pip dependencies
```

Comment:
It looks like it doesn't like this cache because you don't have a `requirements.txt` file anywhere.

I guess maybe you can just delete it.

```suggestion
```